### PR TITLE
[Documentation] Improve README.md in Build section

### PR DESCRIPTION
**Pull request by pacozaa**
According to [this issue](https://github.com/nasa/openmct/pull/903#issuecomment-219785437) 
I've improve the document from above link suggestion.

EDIT1: Check about node app.js and npm start
EDIT2: both command can run the app

### CODE REVIEW

**Comment by pacozaa** [comment]
EDIT1: Check about node app.js and npm start
EDIT2: both command can run the app

**Comment by akhenry** [comment]
@pacozaa looks good to me. I think there are broader issues with README.md that should be addressed, but I have filed a separate issue for that - #1053.

Would you mind adding an author checklist as per the [contributing guide](https://github.com/nasa/openmct/blob/master/CONTRIBUTING.md) then I can go ahead and merge your changes.

Thanks

**Comment by pacozaa** [comment]
Sure!

**Comment by pacozaa** [comment]
@akhenry I have a question about author checklist.

> 2. Unit tests included and/or updated with changes?

Does it mean run `npm test` and get result in `target/tests` with 0 fail?

**Comment by pacozaa** [comment]
Improve documentation from #1053 
This time I try to make a contributing match the rules.
- Edit first paragraph to be the same as official site
- Add official site link.
- Add How is NASA Using Open MCT? Section for attracting people.
- Add Features Section to show the list of Open MCT's capability
- Add See Open MCT in Action Section. Sure! Demo is not only useful but also fun.
- Add Getting Started Section for people who want to play Open MCT in their local machine

### Author Checklist

1. Changes address original issue?Y
2. Unit tests included and/or updated with changes?Y
3. Command line build passes?Y
4. Changes have been smoke-tested?Y

File: README.md [hunk 1]
```diff
@@ -1,7 +1,82 @@
...
+
+##See Open MCT in Action
+
+Do you want to see how Open MCT work? Try our demo.
+[LIVE DEMO](https://openmct-demo.herokuapp.com/)
+![Demo](https://nasa.github.io/openmct/static/res/images/Open-MCT.Browse.Layout.Mars-Weather-1.jpg)
+
+##Getting Started
+
+Here is how to run Open MCT in you local machine.
```
Comment (rcutmore) [inline README.md]: Small typo on this line: 
> Here is how to run Open MCT **on your** local machine.

**Comment by akhenry** [comment]
@pacozaa I really like the direction this is going in. I've made some comments in line.

File: README.md [hunk 2]
```diff
@@ -1,7 +1,82 @@
 # Open MCT
 
-Open MCT is a web-based platform for mission operations user interface
-software.
+Open MCT is a next-generation mission control framework being developed at the NASA Ames Research Center in Silicon Valley. Web-based, for desktop and mobile.
```
Comment (akhenry) [inline README.md]: Can we replace this with the text from the about page on the website -

```
Open MCT is a next-generation mission control framework for visualization of data on desktop and mobile devices. It is developed at NASA's Ames Research Center, and is being used by NASA for data analysis of spacecraft missions, as well as planning and operation of experimental rover systems. As a generalizable and open source framework, Open MCT could be used as the basis for building applications for planning, operation, and analysis of any systems producing telemetry data.
```

File: README.md [hunk 3]
```diff
@@ -1,7 +1,82 @@
...
+
+Please visit our [Official Site](https://nasa.github.io/openmct/)
+
+##How is NASA Using Open MCT?
```
Comment (akhenry) [inline README.md]: Let's remove this section for now. The information is available on the website, and in the readme I want to get straight to how to use it.

File: README.md [hunk 4]
```diff
@@ -1,7 +1,82 @@
...
+
+Software based on Open MCT is being used for mission planning and operations in the lead up to the [Resource Prospector mission](https://www.nasa.gov/resource-prospector) and at NASA's Jet Propulsion Laboratory to view data from the Curiosity Rover.
+
+##Features
```
Comment (akhenry) [inline README.md]: Likewise the features section. Again, I really like the idea and a clear list of features is something that the website is lacking, but I'd preferto leave it out of the readme for now.

File: README.md [hunk 5]
```diff
@@ -1,7 +1,82 @@
...
+ -  Support space missions
+ - Visualize Data
+   - Streaming Data
+   - Historical Data
+   - Imagery
+   - Timelines
+   - Procedures
+   - etc.
+
+##See Open MCT in Action
```
Comment (akhenry) [inline README.md]: Good idea, I like it.

File: README.md [hunk 6]
```diff
@@ -1,7 +1,82 @@
...
+
+Do you want to see how Open MCT work? Try our demo.
```
Comment (akhenry) [inline README.md]: Could we rephrase - To try OpenMCT for yourself, take a look at our online demo.

File: README.md [hunk 7]
```diff
@@ -1,7 +1,82 @@
...
+
+2. Install Development Dependencies
+    `npm install`
+
+3. Run on your local machine
+    `npm start`
+
+4. Open your web browser and go to http://localhost:8080/
+
+5. Wait for splash screen finish then you should be able to play with Open MCT.
```
Comment (akhenry) [inline README.md]: Redundant information, let's remove for now.

File: README.md [hunk 8]
```diff
@@ -1,7 +1,82 @@
...
+
+For more information on building and running Open MCT, please see our getting started guide - https://nasa.github.io/openmct/getting-started/
+
+## Build
```
Comment (akhenry) [inline README.md]: This is confusingly titled (not your fault). This is really describing the steps necessary for deployment and is not necessary for development builds.

Could we rename to 

## Deploying Open MCT

File: README.md [hunk 9]
```diff
@@ -1,7 +1,82 @@
...
+
+Open MCT is built using [`npm`](http://npmjs.com/)
+and [`gulp`](http://gulpjs.com/).
+
+To build:
```
Comment (akhenry) [inline README.md]: Rephrase to 

To build Open MCT for deployment:

File: README.md [hunk 10]
```diff
@@ -1,7 +1,82 @@
...
+The contents of the `dist` folder will contain a runnable Open MCT
+instance (e.g. by starting an HTTP server in that directory), including:
+
+* A `main.js` file containing Open MCT source code.
+* Various assets in the `example` and `platform` directories.
+* An `index.html` that runs Open MCT in its default configuration.
+
+Additional `gulp` tasks are defined in [the gulpfile](gulpfile.js).
+
+### Building Documentation
```
Comment (akhenry) [inline README.md]: Let's make it a peer of "Deploying Open MCT"
ie.

## Building Documentation

File: README.md [hunk 11]
```diff
@@ -1,7 +1,82 @@
...
+
+Open MCT's documentation is generated by an
```
Comment (akhenry) [inline README.md]: Let's mention that the documentation is already available on the website, and add a link to it. Then add a section describing how to build it locally (if people still need to do that)
ie.
### Building the Open MCT Documentation locally

**Comment by pacozaa** [comment]
Thank you @akhenry @rcutmore for the comments.

Let's conclude it as list.

- [ ] 1. Edit typo at line 31. Refer to https://github.com/nasa/openmct/pull/1044#discussion-diff-69346981
- [ ] 2. Edit grant message at line 4. Refer to https://github.com/nasa/openmct/pull/1044#discussion-diff-69635955
- [ ] 3. Remove`How is NASA Using Open MCT?` and `Features` sections. Refer to https://github.com/nasa/openmct/pull/1044#discussion-diff-69636073 and https://github.com/nasa/openmct/pull/1044#discussion-diff-69636219.  I appreciate you like the idea. it's ok to remove it for now
- [ ] 4. Rephrase `See Open MCT in Action ` section to `Try Open MCT now with our live demo.` Refer to https://github.com/nasa/openmct/pull/1044#discussion-diff-69636366
- [ ] 5. Remove `Getting Started` section. Refer to https://github.com/nasa/openmct/pull/1044#discussion-diff-69636658
- [ ] 6. Rename `Build` section to `Deploying Open MCT`. Refer to https://github.com/nasa/openmct/pull/1044#discussion-diff-69636968
- [ ] 7. Repharase `To build` in line 53 to `To build Open MCT for deployment:` Refer to https://github.com/nasa/openmct/pull/1044#discussion-diff-69637051
- [ ] 8. Move `Building Documentation` section above `Deploying Open MCT` and retitle it to `Documentation`.  Refer to https://github.com/nasa/openmct/pull/1044#discussion-diff-69637307
- [ ] 9. Add message mention that documentation is already available on the website with link to it and Add a topic `Building the Open MCT Documentation locally` inside `Documentation`. Refer to https://github.com/nasa/openmct/pull/1044#discussion-diff-69637624

**Comment by pacozaa** [comment]
@akhenry
### Author Checklist

1. Changes address original issue?Y
2. Unit tests included and/or updated with changes?Y
3. Command line build passes?Y
4. Changes have been smoke-tested?Y

File: README.md [hunk 12]
```diff
@@ -1,56 +1,39 @@
...
-Do you want to see how Open MCT work? Try our demo.
-[LIVE DEMO](https://openmct-demo.herokuapp.com/)
+Try Open MCT now with our [live demo](https://openmct-demo.herokuapp.com/).
 ![Demo](https://nasa.github.io/openmct/static/res/images/Open-MCT.Browse.Layout.Mars-Weather-1.jpg)
 
-##Getting Started
+## Documentation
 
-Here is how to run Open MCT in you local machine.
+Check our [Documentation on website](https://nasa.github.io/openmct/documentation/).
```
Comment (akhenry) [inline README.md]: Let's reword slightly. 

    Documentation is available on the [Open MCT website](https://nasa.github.io/openmct/documentation/)'

File: README.md [hunk 13]
```diff
@@ -1,56 +1,39 @@
...
 
-1. Download or Clone Open MCT
- `git clone https://github.com/nasa/openmct.git`
+You can also build it locally.
```
Comment (akhenry) [inline README.md]: Slight rewording.

    The documentation can also be built locally.

**Comment by akhenry** [comment]
@pacozaa This looks really great, thanks for all your work.

There are two minor comments inline. Also, looking at the readme now, I do think we should include instructions for building and running open mct in the readme, as well as linking to the documentation. I think the github site should have all the basic information necessary to at least run Open MCT locally.

So, to that end, could we please add the following section below "See Open MCT In Action". Once we have this in place, I think we're done 👍 

## Building and Running Open MCT Locally

Building and running Open MCT in your local dev environment is very easy. Be sure you have Git and Node.js installed, then follow the directions below. Need additional information? Check out the [Getting Started](https://nasa.github.io/openmct/getting-started/) page on our website.

### 1. Clone the source code
```
git clone https://github.com/nasa/openmct.git
```
### 2. Install development dependencies
```
npm install
```
### 3. Run a local development server
```
npm start
```
Open MCT is now running, and can be accessed by pointing a web browser at http://localhost:8080/

**Comment by pacozaa** [comment]
### Author Checklist

1. Changes address original issue?Y
2. Unit tests included and/or updated with changes?Y
3. Command line build passes?Y
4. Changes have been smoke-tested?Y

File: README.md [hunk 14]
```diff
@@ -1,7 +1,69 @@
...
+
+3. Run a local development server
+
+ `npm start`
+
+Open MCT is now running, and can be accessed by pointing a web browser at http://localhost:8080/
+
+## Documentation
+
+Documentation is available on the [Open MCT website](https://nasa.github.io/openmct/documentation/)
```
Comment (akhenry) [inline README.md]: We need a full stop (period) at the end of this sentence.

**Comment by akhenry** [comment]
Looks great, just one full stop (period) missing. My fault, I missed it in an earlier review. No need to post a new author checklist this time.

**Comment by akhenry** [comment]
@pacozaa Ignore previous comment, I've merged anyway. Thanks for all of your help!

**Pull request merged**